pyEpiabm/pyEpiabm/_csv_writer.py

```diff
@@ -0,0 +1,40 @@
+import csv
+import typing
+
+
+class _CsvWriter:
+    def __init__(self, filename: str, fieldnames: typing.List):
+        """Initialises a file to store output in, and which categories
+        to record.
+
+        :param filename: Output file name
+        :type filename: string
+        :param fieldnames: List of categories to be saved
+        :type fieldnames: list
+        """
+        try:
+            self.f = open(filename, 'w')
+            self.writer = csv.writer(
+                self.f, delimiter=',')
+            self.writer.writerow(fieldnames)
+        except FileNotFoundError as e:
+            self.f = None
+            self.writer = None
+            # TODO: Log file not found error
+            print(f"FileNotFoundError: {str(e)}.")
```

Comment:
I may be misunderstanding - what is the role of printing the error than raising it? If there is a reason do say - I am not sure on the best practice here!

Reply:
Referring to Nick but I think something about specifying which file the user should look for if they get this error

Reply:
@NicholasFan235 can maybe expand on this, but that makes sense, and it's all slightly temporary anyway until we finish the logger anyway!

Reply:
Should be a raise rather than print. I added the comment that we should log details of what file was not found, but that shouldn't be through a print.